devlab/tests/test_resource_migration.py

```diff
@@ -19,6 +21,16 @@ def validate_resource_parameter_in_dst(self, src_list, dst_list,
             'Not all %ss migrated correctly' % resource_name
         )
 
+    def validate_resource_parameter_not_in_dst(self, src_list, dst_list,
+                                               resource_name='resource',
+                                               parameter='name'):
+        # Validating only uniq parameter's value
+        self.assertFalse(
+            set([x.__dict__[parameter] for x in src_list]).issubset(
+                set([x.__dict__[parameter] for x in dst_list])),
+            'Not all %ss migrated correctly' % resource_name
```

Comment:
Can we also have a list of items which did not migrate? E.g.:
```
source_resources = set([x.__dict__[parameter] for x in src_list])
dest_resources = set([x.__dict__[parameter] for x in dst_list])
if not source_resources <= dest_resources:
     missing = source_resources - dest_resources
     self.fail("Not all {resources} migrated. Missing resources on dest: {missing}".format(resources=resource_name, missing=missing)
```
This would greatly improve readability of failure reports